src/invidious/helpers/crystal_class_overrides.cr

```diff
@@ -18,6 +18,40 @@ end
 class HTTP::Client
   property family : Socket::Family = Socket::Family::UNSPEC
 
+  # Override stdlib to automatically initialize proxy if configured
+  #
+  # Accurate as of crystal 1.10.1
```

Comment:
Does it work with other crystal versions?

Reply:
Yep the initialize function wasn't touched